docs/rules/quote-props.md

```diff
@@ -216,9 +216,35 @@ and the following patterns _stop_ being problems:
 ```js
 var x = {
     "100": 1
+};
+```
+
+A `computed` flag, with default value `false`, can also be used as a modifier for the `"as-needed"` mode. When it is set to `true`, unnecessary property name computations will be reported.
```

Comment:
Hmm, not sure `computed` is clear enough, as it's quite different from `numbers`, which require numeric properties to be quoted. I'd expect `computed` to then mean that computed properties should be quoted...but that's not what it does.

Maybe `allowComputed` with a default of `true`, which means don't check computed properties at all.

Eh, you know, I'm starting to think I was wrong about bundling this functionality with `quote-props`. :(